features/select_attribute.feature

```diff
@@ -0,0 +1,12 @@
+#language: fr
+
+Fonctionnalité: Sélectionner des éléments en fonction d'un attribut donné
+
+Contexte:
+
+Soit "vitraux" le portfolio ouvert
+
+Scénario: Recherche par attribut
+
+  Quand on met "PSM 012" dans la barre de recherche et on clique sur "PSM 012"
```

Comment:
Please avoid defining the user interface in scenarios (do that only in step definitions).

Reply:
Does your code works only for names or also for other attributes?
If it does work for other attribute, please add another scenario or replace this scenario with a more common scenario.